src/logics/syncswap/configs.ts

```diff
@@ -0,0 +1,52 @@
+import * as common from '@protocolink/common';
+import { zksyncTokens } from './tokens';
+
+export type ContractNames = 'PoolMaster' | 'ClassicPoolFactory' | 'StablePoolFactory' | 'Router' | 'RouteHelper';
+
+export interface Config {
+  chainId: number;
+  contract: Record<ContractNames, string>;
+  baseTokenAddresses: string[];
+  tokens: common.Token[];
+}
+
+export const configs: Config[] = [
+  {
+    chainId: common.ChainId.zksync,
+    contract: {
+      PoolMaster: '0xbB05918E9B4bA9Fe2c8384d223f0844867909Ffb',
+      ClassicPoolFactory: '0xf2DAd89f2788a8CD54625C60b55cD3d2D0ACa7Cb',
+      StablePoolFactory: '0x5b9f21d407F35b10CbfDDca17D5D84b129356ea3',
+      Router: '0x2da10A1e27bF85cEdD8FFb1AbBe97e53391C0295',
+      RouteHelper: '0x5C07E74cB541c3D1875AEEE441D691DED6ebA204',
```

Comment:
I dig it from the internet connection of the official website. And I asked the official engineer to confirm that this address is correct.

Reply:
Do you have the src code link of RouterHelper? It's hard to verify whether the interface is correct because the contract is not verified on the zkSync mainnet.